examples/moveit.py

```diff
@@ -65,6 +65,7 @@ def main():
             screen.blit(o.image, o.pos)
 
         pg.display.update()
```

Comment:
ps. Since update() is called with no arguments, it is equivalent to flip. So probably this should have display.flip here.

(I'm not saying anyone should necessarily make this fix, just something I noticed.)